applications/poisson/sine/application.h

```diff
@@ -269,10 +263,30 @@ class Application : public ApplicationBase<dim, n_components, Number>
           unsigned int const frequency   = 2;
           apply_deformed_cube_manifold(tria, left, right, deformation, frequency);
         }
+        else if(mesh_type == MeshType::BoundaryLayer)
+        {
+          dealii::Tensor<1, dim> dimensions;
+          for(unsigned int d = 0; d < dim; ++d)
+          {
+            dimensions[d] = right - left;
+          }
+
+          double const grid_stretch_factor = 2.8;
+
+          static BoundaryLayerManifold<dim> manifold(dimensions, grid_stretch_factor);
```

Comment:
I think we do not need to make the manifolds static any more (that used to be needed before the manifold redesign in 2019/2020 I believe). So to limit the number of static variables, I would suggest to write
```suggestion
          BoundaryLayerManifold<dim> manifold(dimensions, grid_stretch_factor);
```
(I realize there are other places, too, but I guess we can start working in that direction.)